Client/Coordinators/Browser/BrowserNavigationHandler.swift

```diff
@@ -16,4 +16,17 @@ protocol BrowserNavigationHandler: AnyObject {
     ///
     /// - Parameter homepanelSection: The section to be displayed.
     func show(homepanelSection: Route.HomepanelSection)
+
+    /// Shows the share extension.
+    ///
+    /// - Parameter url: The url to be shared.
+    /// - Parameter sourceView: The reference view to show the popoverViewController.
+    /// - Parameter toastContainer: The view in which is displayed the toast results from actions in the share extension.
```

Comment:
nit, Just realized we're missing `popoverArrowDirection` parameter in the doc

Reply:
ok i'll add it now